packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,29 @@
+## 45.5.0 - March 7, 2019
+
+### Added
+
+#### salesforcedx-vscode-apex
+
+- Hover capability to show Apex Javadoc information when hovering on symbols ([PR #1106](https://github.com/forcedotcom/salesforcedx-vscode/pull/1106))
+
+### Fixed
+
+#### salesforcedx-vscode-docs
```

Comment:
@ruthemmanuelle now that we moved the docs to github pages I was thinking about having a dedicated section for it on the change log, what do you think ?

Reply:
Sounds great. Maybe it should be `salesforcedx-vscode/docs`, though? I'm suggesting that because the docs aren't their own extension, so they don't have a Marketplace page with a URL that includes `salesforcedx-vscode-docs`, but their files live in `salesforcedx-vscode/docs` in the repo.

Reply:
Yeah, changing it to `salesforcedx-vscode/docs` makes sense 👍

Reply:
Cool, I'll make that change along with some other edits.